src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioActiveDocumentTracker.cs

```diff
@@ -65,7 +65,7 @@ internal class VisualStudioActiveDocumentTracker : ForegroundThreadAffinitizedOb
             {
                 await ThreadingContext.JoinableTaskFactory.SwitchToMainThreadAsync();
 
-                var monitorSelectionService = (IVsMonitorSelection)await asyncServiceProvider.GetServiceAsync(typeof(SVsShellMonitorSelection)).ConfigureAwait(false);
+                var monitorSelectionService = (IVsMonitorSelection)await asyncServiceProvider.GetServiceAsync(typeof(SVsShellMonitorSelection)).ConfigureAwait(true);
```

Comment:
Is this worth a comment in case a future developer thinks its a typo?

Reply:
The guidelines and analyzers explicitly request this, so I imagine there's an IDE squiggle for this?

https://github.com/microsoft/vs-threading/blob/master/doc/cookbook_vs.md#short-answer

https://github.com/microsoft/vs-threading/blob/master/doc/analyzers/VSTHRD111.md#vsthrd111-use-configureawaitbool

Reply:
> Is this worth a comment in case a future developer thinks its a typo?

No, this is a global rule for the product, as opposed to a special case.